libs/designsystem/src/lib/scss/print/utilities/_no-print.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
```

Comment:
Is this intended to be used by target projects to slap onto elements they do not want to be shown when printed? 

If that is the case I'm a bit iffy whether it should live in Kirby. I guess it might as well live in the consumer project as there really is nothing Kirby specific to it.